utils/utils_test.go

```diff
@@ -0,0 +1,32 @@
+package utils
+
+import (
+	"os"
+	"reflect"
+	"testing"
+)
+
+func TestGetOpt(t *testing.T) {
+	// If EnvVar has a value, GetOpt should return it.
+	expected := "bar"
+	err := os.Setenv("TEST_FOO", expected)
+	if err != nil {
+		t.Error(err)
+	}
+
+	actual := GetOpt("TEST_FOO", "baz")
+	if !reflect.DeepEqual(actual, expected) {
+		t.Errorf("Expected %s, but got %s", expected, actual)
+	}
+
+	// If EnvVar has an empty value, GetOpt should return the default value.
+	err = os.Setenv("TEST_EMPTY", "")
+	expected = "foobar"
+	if err != nil {
+		t.Error(err)
+	}
+	actual = GetOpt("TEST_EMPTY", expected)
+	if !reflect.DeepEqual(actual, expected) {
+		t.Errorf("Expected %s, but got %s", expected, actual)
+	}
+}
```

Comment:
Something to consider... sometime between router 2.0.0 and now, we've started using https://github.com/kelseyhightower/envconfig in lots of our projects for managing configuration.

👍 on adding these tests, obviously, but I think a follow up PR to _remove_ the custom env var parsing would be 💯 . 😄